portability-spi-transfer/src/main/java/org/datatransferproject/spi/transfer/idempotentexecutor/InMemoryIdempotentImportExecutorExtension.java

```diff
@@ -14,6 +14,11 @@ public IdempotentImportExecutor getIdempotentImportExecutor(ExtensionContext ext
     return new InMemoryIdempotentImportExecutor(extensionContext.getMonitor());
   }
 
+  @Override
```

Comment:
it looks liek the build is failing because this something is missing this implementation, i think we need to mark this as @default

Reply:
I think so too.

Reply:
I realized I forgot to add the getRetryingIdempotentImportExecutor to IdempotentImportExecutorExtension and it fixed the build issues, should I still mark this @default?

Reply:
Ah I see -- so i think you need a default implementation in IdempotentImportExecutorExtension.java -- that way the specific implementations (GoogleCloudIdempotentImportExecutor and InMemoryIdempotentImportExecutor and others that might not be in github) don't need to add an override for that implementation.